texting/migrations/0007_reminder_err_code.py

```diff
@@ -9,33 +9,60 @@ class Migration(migrations.Migration):
 
     dependencies = [
         migrations.swappable_dependency(settings.AUTH_USER_MODEL),
-        ('texting', '0006_alter_phonenumberlookup_carrier'),
+        ("texting", "0006_alter_phonenumberlookup_carrier"),
```

Comment:
I suspect you saved this at some point and it triggered formatting by black, but we don't normally auto-format migrations w/ black, so maybe revert this file to make the PR easier to understand?  Eh, up to you.

Reply:
yeah, it thought about it and since in this one, there's that long list of choices that otherwise is one super long line, I wanted to leave this one formatted.